incubator/binding-grpc-kafka/src/main/java/io/aklivity/zilla/runtime/binding/grpc/kafka/internal/config/GrpcKafkaWithProduceResult.java

```diff
@@ -148,6 +149,16 @@ private void method(
 
     private void correlationId(
         KafkaHeaderFW.Builder builder)
+    {
+        builder
+            .nameLen(correlation.replyTo.length())
+            .name(correlation.replyTo.value(), 0, correlation.replyTo.length())
+            .valueLen(replyTo.length())
+            .value(replyTo.value(), 0, replyTo.length());
+    }
+
+    private void replyTo(
```

Comment:
I think the methods `correlationId` and `replyTo` might need to be switched based on their implementation.